src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="current-directors">
+
+    <v-card flat>
+      <!-- Header -->
+      <div class="header-container">
+        <v-icon color="app-dk-blue">mdi-account-multiple-plus</v-icon>
+        <label class="font-weight-bold pl-2">Current Directors</label>
+      </div>
+
+      <!-- Instructional Text -->
+      <div class="instructional-text pt-10 pb-15 px-4">
+        To change directors, please use the Change feature in the Current Directors list on you business dashboard.
+      </div>
+
+      <div class="list-container px-4">
+        <!-- List Display Section -->
+        <div id="people-roles-list" v-if="currentDirectors.length > 0">
+          <!-- List Headers -->
+          <v-row class="people-roles-list-header list-item__subtitle pb-3" no-gutters>
+            <v-col class="pr-5" v-for="(title, index) in tableHeaders" :key="index">
+              <span class="directors-title">{{ title }}</span>
+            </v-col>
+          </v-row>
+
+          <!-- List Content -->
+          <v-row
+            class="people-roles-content py-3"
+            v-for="(orgPerson, index) in currentDirectors"
+            :key="index"
+            no-gutters
+          >
+            <!-- Name + Badge -->
+            <v-col class="text-truncate pr-5">
+              <!-- provide tooltip to display full name if name is longer than 25 chars -->
+              <v-tooltip top :disabled="formatFullName(orgPerson.officer).length < 25" color="primary">
+                <template v-slot:activator="{ on }">
+                  <span v-on="on" class="director-name">{{ formatFullName(orgPerson.officer).toLowerCase() }}</span>
```

Comment:
Changing director name to lower case doesn't seem right. Do you have a spec/design that requests this?

Reply:
I have css that changes it to capitalize. The stored data, at least that I was testing with was in all caps so this was to get it to match the design https://projects.invisionapp.com/share/6EYT1CDGYVN#/screens/445576065/comments which shows it as capitalized

Reply:
So the design shows it as title caps.

The sample data you have may be all-caps from the db... but don't assume that and convert it.

If you have CSS that capitalizes it, why are you converting it to lower case here?

Bottom line: display what you get from the db/API. Otherwise you're likely to get it wrong, eg. "McDonald" don't display right if you title-case it. That might cause a user to want to fix it, but it would still display wrong.

Reply:
I was converting to lowercase because CSS capitalize doesn't convert the rest of the word to lower, but I see your point here, I will remove that and trust the db

Reply:
If, some other time, you do need to fully uppercase a word using CSS, you can use `text-transform: uppercase;`.